main.md

```diff
@@ -929,6 +929,17 @@ the authorization server MUST reject the request.
 The rejection response MAY include a `DPoP-Nonce` HTTP header
```

Comment:
Seems like the "If the nonce claim in the DPoP proof of a token request does not exactly match the nonce supplied by the authorization server to the client, the authorization server MUST reject the request." should be loosened up or qualified just a bit too.

Reply:
This is immediately followed by "The rejection response MAY include a `DPoP-Nonce` HTTP header providing a new nonce value to use for subsequent requests."  I believe that's the qualification you're looking for.

Was there something else you had in mind in addition to this?